tests/unit/amo/components/TestSearchResult.js

```diff
@@ -64,35 +64,50 @@ describe(__filename, () => {
   it('renders the heading', () => {
     const root = render();
 
-    expect(root.find('.SearchResult-link').children()).toIncludeText(
+    expect(root.find('.SearchResult-title').children()).toIncludeText(
       'A search result',
     );
   });
 
-  it('links the heading to the detail page', () => {
+  it('links the card to the detail page', () => {
     const slug = 'some-addon-slug';
     const addon = createInternalAddon({ ...fakeAddon, slug });
 
     const root = render({ addon });
 
-    expect(root.find('.SearchResult-link')).toHaveProp('to', getAddonURL(slug));
+    expect(root.find('.SearchResult-wrapper')).toHaveProp(
+      'to',
+      getAddonURL(slug),
+    );
+    expect(root.find('.SearchResult-wrapper')).not.toHaveProp('href', '#');
```

Comment:
This should just be `expect(root.find('.SearchResult-wrapper')).not.toHaveProp('href');`. I.e., we should check that it doesn't have an `href` prop at all, as opposed to checking that it doesn't have an `href` prop with a particular value.